kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/security/ElytronUserGroupAdapter.java

```diff
@@ -36,14 +43,56 @@
         if (identityOptional.isPresent()) {
             SecurityIdentity identity = identityOptional.get();
             if (identity.getPrincipal().getName().equals(userId)) {
```

Comment:
```suggestion
            if (userId.equals(identity.getPrincipal().getName())) {
```
To avoid NPE

Reply:
it does not really matter if you think about it but done anyway